outbound/index.js

```diff
@@ -313,29 +313,27 @@ exports.process_delivery = function (ok_paths, todo, hmails, cb) {
 };
 
 exports.build_todo = function (todo, ws, write_more) {
-    // Replacer function to exclude items from the queue file header
-    function exclude_from_json (key, value) {
-        switch (key) {
-            case 'message_stream':
-                return undefined;
-            default:
-                return value;
-        }
-    }
 
-    const todo_json = JSON.stringify(todo, exclude_from_json)
```

Comment:
Let's just pass `'\t'` as the third param to JSON.stringify instead.

Reply:
That doesn't do nearly the same thing. Although, passing in a `\t` as the third option does make the queue file that even more human friendly, so I'm happy to do that too. The `\n`'s put the first Received header onto a newline, after the JSON header.

Reply:
I know what it does, but I figured if we want human readable, the \t makes it so much better.